.github/workflows/testing.yml

```diff
@@ -20,6 +20,8 @@ env:
   # Some tests require the 4C executable to check if the created input file
   # works with 4C.
   MESHPY_FOUR_C_EXE: /home_local/github-runner/testing_lib/four_c_master/release/4C
+  # Specify the testing directory
```

Comment:
Let's put this change to the testing workflow into an own pull request.

Can we avoid adding the new parameter `MESHPY_TESTING_DIR`? As far as I know, we install meshpy into `${{ github.workspace }}`. Then we should be able to specify the path to the testing directory as `${{ github.workspace }}/tests/testing-tmp/`.

Reply:
Yes, I wanted to do this initially and just test it here.